### DISCUSSION TITLE
is it possible cutting and dividing object or line?

### DISCUSSION THREAD

**Discussion by KiDeokKim** [fabricjs/fabric.js#8216] [Q&A] [answered]
(https://helpx.adobe.com/illustrator/using/cutting-dividing-objects.html)

i read above URL, and checked **scissors part**

so i want cutting the object(like rectangle, polygon or line) at the selected objects point,

i don't know how to develope this function.

please help me if you have any ideas.

**Comment by ShaMan123** [comment, marked as answer]
This is not an issue so next time post under discussions please

It seems that clipPath (maybe inverted) will work using Intersection to generate the shape that will clip the original one

**Reply by ShaMan123** [reply]
Look into util.sendObjectToPlane
It will help you
